File: 35-SearchInsertPosition/35-SearchInsertPosition.java
// Last updated: 7/13/2026, 10:15:55 AM
class Solution {
    public int searchInsert(int[] nums, int target) {
        int i=0,val=0;
        while(i<nums.length){
            if(nums[i]<=target) {
                if (nums[i]==target) return i;
                else val=i+1;
            }
            i++;
        }
   return val; }
}

File: 35-SearchInsertPosition/35-SearchInsertPosition.py
# Last updated: 7/13/2026, 10:16:03 AM
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        if target in nums:
            return nums.index(target)
        else:
            if target not in nums:
                nums.append(target)
                p=sorted(nums)
                return p.index(target)